packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -158,4 +174,10 @@ export class SDKClient {
             ? input.substring(2)
             : input;
     }
+
+    private static toTinyBar(balance: AccountBalance): BigNumber {
```

Comment:
the code that follows seems to go from hbar -> tiny bar -> weibar but the function name is toTinyBar. Am I missing something simple?